src/inject.js

```diff
@@ -48,9 +48,9 @@ const stringHandlers = {
     // an array of objects and strings.
     fontFamily: function fontFamily(val) {
         if (Array.isArray(val)) {
-            return val.map(fontFamily).join(",");
+            return uniquify(val.map(fontFamily)).join(",");
```

Comment:
I don't think it's terribly important to uniquify these font names. It'll still work without this, correct?